src/components/Tabs/TabsNav.test.tsx

```diff
@@ -27,30 +28,101 @@ afterEach(() => {
   jest.clearAllMocks();
 });
 
-const renderComponent = (props?: TabsNavProps) => render(<TabsNav {...DEFAULT_PROPS} {...props} />);
+describe('<TabsNav />', () => {
+  describe('class names', () => {
+    test('should apply Container class only to the container element', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const tabNavContainer = container.getElementsByClassName(styles.Container);
 
-test('should render react-router-dom Link elements with correct class names', () => {
-  const { getAllByTestId } = renderComponent();
-  const links = getAllByTestId('link');
-  expect(links.length).toEqual(MOCK_TABS.length);
-  links.forEach(l => expect(l.classList).toContain('Tab'));
-});
+      expect(tabNavContainer.length).toBe(1);
+    });
 
-test('should apply Selected class name on selected Link', () => {
-  const { container } = renderComponent();
-  const selected = container.getElementsByClassName('Selected');
-  expect(selected.length).toBe(1);
-  expect(selected[0].textContent).toBe(MOCK_TABS[0].text);
-});
+    test('should apply List class only to the List element', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const list = container.getElementsByClassName(styles.List);
 
-test('should apply replace prop to Link', () => {
-  renderComponent({ tabs: [MOCK_TABS[0]] });
-  expect(mockLink).toHaveBeenCalledTimes(1);
-  expect(mockLink).toHaveBeenCalledWith(expect.objectContaining({ replace: true }));
-});
+      expect(list.length).toBe(1);
+    });
+
+    test('should apply Tab class only to the Link elements', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const link = container.getElementsByClassName(styles.Tab);
+      expect(link.length).toBe(3);
+    });
+
+    test('should apply Selected class on default selected link', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const selectedLink = container.getElementsByClassName(styles.Selected);
+
+      expect(selectedLink[0]).toHaveTextContent(MOCK_TABS[0].text);
+    });
+
+    test('should apply Selected class only to the selected link', () => {
+      const { container } = render(<TabsNav {...DEFAULT_PROPS} />);
+      const selectedLink = container.getElementsByClassName(styles.Selected);
+
+      expect(selectedLink.length).toBe(1);
+      expect(selectedLink[0]).not.toHaveTextContent(MOCK_TABS[1].text);
+      expect(selectedLink[0]).not.toHaveTextContent(MOCK_TABS[2].text);
```

Comment:
Change to `expect(selectedLink.textContent).toBe(...)`.

Reply:
Updated - noted to be aware of this one moving forward - also changed in a couple of other places in these tests.